src/components/QuestionEdit/index.tsx

```diff
@@ -144,7 +144,7 @@ export default function QuestionEdit({
 
   function onDeleteClick(ev) {
     ev.preventDefault();
-    deleteQuestion();
+    deleteQuestion(ev);
```

Comment:
Why this change?

Reply:
we are using ev.stopPropagation in deleteQuestion function. it was throwing console error